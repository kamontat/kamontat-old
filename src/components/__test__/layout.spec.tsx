import React from "react";
import { shallow } from "enzyme";

import Layout from "../layout";

describe("Layout component", () => {
  it("should set container children element to data", () => {
    const wrapper = shallow(<Layout>data</Layout>);
    expect(wrapper).toMatchSnapshot();
  });
});
