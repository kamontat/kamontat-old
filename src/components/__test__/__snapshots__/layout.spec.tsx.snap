// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should set container children element to data 1`] = `<div />`;
